README.md

```diff
@@ -99,6 +99,7 @@ COMMANDS
       java          generate the models for Java
       javascript    generate the models for JavaScript
       dart          generate the models for Dart
+    fromTemplate     generate whatever you want using templates compatible with AsyncAPI Generator 
```

Comment:
Maybe you can remove the doc part 

```
As of now, the @asyncapi/cli only supports validation of the specification file. (This is still under development for more features.)
```
Because now the CLI support other features

Reply:
It seems that this line of docs is not removed :)